src/inc/common/Mlp_Helpers.php

```diff
@@ -479,7 +479,7 @@ public static function show_linked_elements( $args ) {
 
 		$output .= '</ul></div>';
 
-		return $output;
+		return apply_filters( 'show_linked_elements', $output, $items, $params );
```

Comment:
I think this filter name should have a prefix, to avoid clashes with other plugins which might have the exact same hook name.